addons/binding/org.openhab.binding.network/ESH-INF/thing/thing-types.xml

```diff
@@ -1,73 +1,119 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<thing:thing-descriptions bindingId="network"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
-        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd"><!--Network Binding-->
-    <thing-type id="device">
-        <label>Network Device</label>
-        <description>Device which must be connected to the same network</description>
+<thing:thing-descriptions bindingId="network" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd"><!--Network Binding -->
+	<thing-type id="pingdevice">
```

Comment:
It is only a breaking change if someone used the port parameter, because I map the thingID "device" to "pingdevice" in the handler factory. And because we had two issues about the non working port parameter it is actually not a breaking change at all.